docs/.vitepress/theme/custom.css

```diff
@@ -129,6 +159,225 @@ html.dark body {
 {
   color: var(--vp-c-brand-2);
 }
+.VPHero {
+  --vp-c-bg: #0F1728;
+  background-color: var(--vp-c-bg);
+  margin-bottom: 60px;
+  position: relative;
+}
+.VPHero::before {
+  content: '';
+  position: absolute;
+  bottom: 80px;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 104px;
+  background: url('stars-background.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.VPHero::after {
+  content: '';
+  position: absolute;
+  bottom: -8px;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 8px;
+  background: url('hero-wavy-line.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
It looks a bit weird on large screens :D 
![Screenshot 2024-09-23 at 08 46 06](https://github.com/user-attachments/assets/1827112d-17a6-46dc-8f5e-a0a004234be4)

Reply:
I'll look into it! :)

Reply:
How wide was this screen that you noticed this? :) I just checked now on my screen that is 1920px and the waves don't look weird there: 
<img width="1905" alt="Skärmavbild 2024-10-01 kl  15 19 23" src="https://github.com/user-attachments/assets/1874ea91-407d-4ff3-8847-939333a6250d">

Reply:
Ah I see it now! Hmm I wonder how I can adjust the svg-file? 🤔 I noticed that the bigger the screen is (from 2000 and up), the straighter the wave gets. 🙈  For example, it looks like this when I change to screen size 3840x2160: 
![Skärmavbild 2024-10-01 kl  16 47 53](https://github.com/user-attachments/assets/5526a5e5-875d-47e2-99c3-5c47c4284fdb)

Reply:
Not sure this is the way to go, but when I removed width, height and viewbox from the `hero-wavy-line.svg`, the problem disappeared 😄

Reply:
hard to tell from screenshots since the scaling is not 1:1, isnt it just vertically cut on the lower edge?

Reply:
<img width="919" alt="image" src="https://github.com/user-attachments/assets/28d54304-6503-49dd-9c97-e8c781fcde70">
This ::after sometimes render lower than the graphic.